decidim_app-design/app/assets/stylesheets/_initiatives.scss

```diff
@@ -0,0 +1,32 @@
+.initiatives-form{
```

Comment:
I understand sometimes it's difficult to sort out - I'm no expert so I trust your gut. That said, it looks like the `.initiative-adhesion-list` could be factored out, at the minimum least.

Reply:
I've done a [component/wrapper](https://github.com/decidim/decidim/pull/2466/commits/c1862db963edf0fb4c15e99839bc7ed860daad51#diff-25b5703e6e96471051121c2f369bd669), I don't know how it call it accurately, to use it instead of the `adhesion-list`. So, that content is removed from the SCSS file. Take a look.